app/scripts/controllers/rif/configuration/index.js

```diff
@@ -163,13 +168,13 @@ export class RifConfigurationProvider {
   }
 
   getConfigurationObject () {
-    return this.store.getState().rifConfiguration;
+    return this.store.getState().configuration[this.network ? this.network.id : '33'];
```

Comment:
33 is not so standard for regtest. Lets put that number into a constant so its configurable

Reply:
Ok didn't knew that, good to know